### ISSUE TITLE
RunnerDeployment not marking correct CURRENT and READY values

### ISSUE THREAD

**Issue by AlexDHoffer** [actions-runner-controller/actions-runner-controller#490]
Hi folks, excellent work with this project, I am having a great time using it so far.

One thing I'm very curious about - I have a RunnerDeployment defined like so:

````yaml
- apiVersion: actions.summerwind.dev/v1alpha1
  kind: RunnerDeployment
  metadata:
    name: runner-deploy
    namespace: actions-runner-system
  spec:
    replicas: 2
    selector:
      matchLabels:
        runner-deployment-name: runner-deploy
    template:
      spec:
        dockerdContainerResources: {}
        image: ""
        labels:
        - k8s-runner
        - ubuntu
        organization: [my-org]
        resources:
          requests:
            cpu: 10m
````

The pods are running:

![image](https://user-images.githubusercontent.com/19369483/116465092-9f019400-a821-11eb-94ff-e90e232b7bb1.png)

But the runner deployment does not report the correct values:

![image](https://user-images.githubusercontent.com/19369483/116465152-b2146400-a821-11eb-99fe-9bb92ba287fe.png)

I may be missing something simple, but I'd like to have those numbers reflected there. Thanks!

**Comment by callum-tait-pbx** [comment]
Same behaviour for RunnerDeployments backed by a HRA, from reading about `additionalPrinterColumns` I'm not sure those 2 columns are actually implemented in the code.

**Comment by mumoshu** [comment]
This is going to be resolved via #556 shortly.
